src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -625,6 +625,14 @@ uct_gga_mlx5_iface_is_reachable_v2(const uct_iface_h tl_iface,
            uct_ib_iface_is_reachable_v2(tl_iface, params);
 }
 
+static int
+uct_gga_mlx5_ep_is_connected(uct_ep_h tl_ep,
+                             const uct_ep_is_connected_params_t *params)
+{
+    return uct_rc_mlx5_base_ep_is_connected(tl_ep, params) &&
+           uct_base_ep_is_connected(tl_ep, params);
```

Comment:
why need uct_base_ep_is_connected? seems rc_mlx5 does not need it

Reply:
to compare iface address (only required for GGA)

Reply:
both will compare IB device address which is redundant
can we reuse common code with uct_gga_mlx5_iface_is_reachable_v2 and compare iface address directly?
BTW @evgeny-leksikov @amaslenn the gga transport does not currently implement returning info string from uct_gga_mlx5_iface_is_reachable_v2

Reply:
> BTW @evgeny-leksikov @amaslenn the gga transport does not currently implement returning info string from uct_gga_mlx5_iface_is_reachable_v2

thanks, I'll add